### ISSUE TITLE
Confusion between short and long-form model name

### ISSUE THREAD

**Issue by mathieuboudreau** [axondeepseg/axondeepseg#831]
### Discussed in https://github.com/axondeepseg/axondeepseg/discussions/829

<div type='discussions-op-text'>

<sup>Originally posted by **bigzero61** September 26, 2024</sup>
ADS almost completely satisfies my need in TEM analysis. But there are a little mistakes in the 'existing models' section in the documentation that may confuse beginner. 
```
To download these models, you must first have AxonDeepSeg installed. Afterwards, run:

download_model -m <model name> -t <model type>
where <model name> is the full name (e.g. model_seg_rat_axon-myelin_SEM) and <model type> is either light or ensemble. To view available models and their details, run:

download_model --list
```

The command should be 'download_models'. And the the example for the <model name> is misleading. The full model name shown in the model list encounters key error in this command (e.g. 'unmyelinated-TEM' works instead of 'model_seg_unmyelinated_sickkids').

Besides, in the 'Using AxonDeepSeg' section, it should be noted that the 'ads_activate' command should be ran first to activate the environment before using 'axondeepseg' command.

Lastly, thank AxonDeepSeg and its contributors! This software did save a lot of my time!</div>

@hermancollin pinging you for your thoughts - is having the long-form model name listed as "model name" essential here, or could we have this information in another tag?

**Comment by hermancollin** [comment]
Yes I totally agree that this is a problem - luckily this user caught it! 

I think we should only keep the long format. The users will have to specify the long format + the model type suffix everytime they want to use ADS with the `-m` option (because the model folder name is made from these 2 fields), so it makes sense to me to have them type it out when they want to download it. However, the keys we currently use are arbitrary and misleading. I suggest we remove the "name" field from the `MODELS` dict and use it directly as the key.

https://github.com/axondeepseg/axondeepseg/blob/35d783c8edbb75f19f6c2ae33b2a30043252b4fa/AxonDeepSeg/model_cards.py#L5-L17

**Comment by mathieuboudreau** [comment]
>. The users will have to specify the long format + the model type suffix everytime they want to use ADS with the -m option (because the model folder name is made from these 2 fields),

Is there a reason why we insist on using this long format for the directory names and not the short form name? Would there be some confusion if only the short-form name was set as the foldername when the models are installed?

**Comment by hermancollin** [comment]
Not really. The reason I always use this long format name is because of the old ivadomed formatting (see https://github.com/ivadomed/ivadomed/wiki/repositories#models), but now that you mention it we haven't really discussed this since 2021.

> Is there a reason why we insist on using this long format for the directory names and not the short form name? Would there be some confusion if only the short-form name was set as the foldername when the models are installed?

This is true. The model directory could be renamed during the installation step and we could completely "hide" this long format convention from the user. This is also a valid option.

**Comment by hermancollin** [comment]
An additionnal problem with the current model download script is that we are exposing to the user the "model type" (i.e. ensembling N-folds vs 1-fold). 

I think the user should not have to specify this. Instead, we should either 
- have a default "type" to download
- refactor the **model_cards.py** to only include one URL for every model

Either way, the model download should be refactored and more user-friendly. I'm responsible for this mess, so I'll do my best to clean this up.
